pyproject.toml

```
@@ -58,6 +58,15 @@ path = "data_safe_haven/version.py"
 [tool.hatch.envs.default]
 pre-install-commands = ["pip install -r requirements.txt"]
 
+[tool.hatch.envs.docs]
```

```suggestion
[tool.hatch.envs.docs]
pre-install-commands = ["pip install -r docs/requirements.txt"]
```

To make sure you have the pinned dependencies like for the default env.

---

Sure, but `docs` doesn't currently have any commands in it. Let me add some!

---

Done in 5a03056

---

Even better :+1:.

That's actually a big advantage of using an env. `hatch run docs:build` instead of making/sourcing a venv then running make.